dev-tools/create_bwc_index.py

```diff
@@ -58,28 +58,32 @@ def assert_sort(hits):
 
 # Indexes the given number of document into the given index
 # and randomly runs refresh, optimize and flush commands
-def index_documents(es, index_name, type, num_docs):
+def index_documents(es, index_name, type, num_docs, supports_dots_in_field_names):
```

Comment:
Just a thought for the future (this is fine for now): so these arg lists don't get too large, maybe we could pass version through and let the index method have conditions based on that, like we do when generating mappings.